Makefile

```diff
@@ -245,7 +245,7 @@ train_gpt2: train_gpt2.c
 test_gpt2: test_gpt2.c
 	$(CC) $(CFLAGS) $(INCLUDES) $(LDFLAGS) $^ $(LDLIBS) $(OUTPUT_FILE)
 
-$(NVCC_CUDNN): llmc/cudnn_att.cu
+$(NVCC_CUDNN): llmc/cudnn_att.cpp
```

Comment:
@ngc92 - this broke the windows CUDNN build - not sure why - changing it back fixes it.

Reply:
Fixed it - need this at the top of the source file: #define NOMINMAX